kind.go

```diff
@@ -3,4 +3,7 @@ package awstpr
 const (
 	// Kind is the TPR kind for awstpr resources.
 	Kind = "Aws"
+
+	// PluralKind is the plural TPR kind for use in resource URLs.
+	PluralKind = "Awses"
```

Comment:
It is lower case. See https://github.com/giantswarm/kubernetesd/pull/124/files#diff-2cc6acb4e968e9a08aa598bee2e8fa92L26.